types/istanbul-lib-instrument/package.json

```diff
@@ -18,6 +19,10 @@
         {
             "name": "Jason Cheatham",
             "githubUsername": "jason0x43"
+        },
+        {
+            "name": "Travis Zhang",
+            "githubUsername": "travzhang"
```

Comment:
Are you editing this on purpose? This array is only for people who should have approval permissions, and it's very odd for a first-time contributor to add themselves to an existing and popular package.